.readthedocs.yaml

```
@@ -0,0 +1,23 @@
+# .readthedocs.yml
+# Read the Docs configuration file
+# See https://docs.readthedocs.io/en/stable/config-file/v2.html for details
+
+# Required
+version: 2
+
+build:
+  os: ubuntu-20.04
+  tools:
+    python: '3.9'
```

```suggestion
  os: ubuntu-22.04
  tools:
    python: '3.12'
```
Let's use the latest versions so that we get the latest bugfixes and don't have to update as often.